lesson_4.md

```diff
@@ -0,0 +1,69 @@
+# Lesson IV excercise solutions
+
+## 46
+
+1. Dēlectat, fugāmus, liberas.
+
+He pleases, we are routing, you set free.
+
+2. Exspectāmus, portant, dēlectās.
+
+We are waiting, they are carrying, you are please.
+
+3. Dēlectāmus, fugātis, liberant.
+
+We please, you (all) are routing, they free.
+
+4. Liberatne? liberō, portāmus.
+
+Does he set free? I free, we carry.
+
+5. Fīliās incolārum liberant.
+
+(The) daughters of the inhabitants set free.
+
+6. Poētae, fortūna rēginam dēlectat.
+
+The poet's fortune pleases the queen.
+
+7. Spectāmus fīliās fēminae.
+
+We watch the woman's daughter.
+
+8. Filiane Galbae litterās expectat?
+
+Is Galba's daughter waiting for a letter?
+
+9. Etiam tubae nautārum incolās fugant.
+
+Meanwhile, the sailors' horns put the island to flight.
+
+10. Cuius litterās portās?
+
+Whose letters are you carrying?
+
+## 47
+
+1. I am calling together the sailors of the queen.
+
+Convocō nautas reginae.
+
+2. The poets' daughters love the land.
+
+Poetārum fīliae adorant terra.
+
+3. Whose daughter are you setting free?
+
+Cuius fīliam liberō?
+
+4. Do they praise Galba's daughter?
+
+Laudantne fīliam Galbae?
+
+5. We are now waiting for the farmers.
+
+Exspectāmus agricolas.
+
+6. Whom does the fortune of the farmers delight?
+
+Cuius agricolārum fortuna delectat?
```

Comment:
cuius is the genative of quis. "quem" is the accusative which is what is needed here.

I also think it's confusing to put agricolarum before fortuna although it's technically correct.